Jak-Sidious/YummyRecipies
# Chore oop redesign

#### What's this PR do?

1. Restructured folders to meet Django app structure
2. Created the various Test for the recipe, category and User classes
3. Modify the travis.yml file to include coveralls
4. Modify the .travis.yml file to include the respective badges
5. Move tests into separate directory named test

#### Where should the reviewer start?

The reviewer should look into the main folder to find out where the respective files are located 

#### How should this be manually tested?
To manually analyse the respective test for each class, the reviewer should navigate into the test folder/directory and run nosetest followed by either **test_user.py** **test_recipie.py** **test_category.py**

#### Any background context you want to provide?

All tests and classes were built using the TDD approach 

#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
